[32mINFO    [0m root:conftest.py:10 'Firefox' browser is opened.
[32mINFO    [0m root:test_non-existing_course_search.py:13 Program has started at: 2024-02-07 11:43:10.428453
[32mINFO    [0m root:general_functions.py:105 Loaded JSON data: {'url': 'https://courses.ultimateqa.com/collections', 'login': {'email': 'colemanbrooke@example.com', 'password': '(*^2CoWR'}}.
[32mINFO    [0m root:general_functions.py:18 Navigated to URL: 'https://courses.ultimateqa.com/collections'
[32mINFO    [0m root:general_functions.py:76 'Sign In' button is clicked.
[32mINFO    [0m root:welcome.py:13 'Login' page is opened.
[32mINFO    [0m root:general_functions.py:57 'colemanbrooke@example.com' text is entered in name: 'user[email]' input field.
[32mINFO    [0m root:general_functions.py:57 '(*^2CoWR' text is entered in name: 'user[password]' input field.
[32mINFO    [0m root:general_functions.py:76 'Sign in' button is clicked.
[32mINFO    [0m root:login.py:26 Logged in successfully!!!
[32mINFO    [0m root:general_functions.py:57 '12345' text is entered in name: 'q' input field.
[1m[31mERROR   [0m root:general_functions.py:44 Error in 'find_elems_in_dom': Message: 

[32mINFO    [0m root:test_non-existing_course_search.py:33 No courses found with '12345' key.
[32mINFO    [0m root:test_non-existing_course_search.py:36 Program has ended at: 2024-02-07 11:45:31.950698
[32mINFO    [0m root:test_non-existing_course_search.py:37 Program run duration is: 0:02:21.522245
[32mINFO    [0m root:conftest.py:14 'Firefox' browser is closed.